File: VehicleTransport/主版本/3.开发源代码/Common/Model/NormalCommand/OutCommandModel/OutFlowPathModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Common.Model.OutCommandModel
{
    public class OutFlowPathModel
    {
        /// <summary>
        /// 申请
        /// </summary>
        public  bool Apply = true;

        /// <summary>
        /// 审核
        /// </summary>
        public  bool Check = true;

        /// <summary>
        /// 供车
        /// </summary>
        public bool Give = false;

        /// <summary>
        /// 装车
        /// </summary>
        public  bool Load = true;

        /// <summary>
        /// 交接车
        /// </summary>
        public bool Handover = false;

        /// <summary>
        /// 卸车
        /// </summary>
        public  bool UnLoad = true;

        /// <summary>
        /// 还车
        /// </summary>
        public bool Back = false;
    }
}

File: VehicleTransport/主版本/3.开发源代码/客户端/手机客户端/Last of KJ915_bestway/src/com/bestway/kj915/domain/res/OutFlowPathModel.java
package com.bestway.kj915.domain.res;

import java.util.HashMap;

/**
 * 流程节点控制
 * 
 * @author gaga
 * 
 */
public class OutFlowPathModel extends BasicResInner {

	public boolean Apply;
	public boolean Check;// 不需要用到
	public boolean Give;
	public boolean Load;
	public boolean Handover;
	public boolean UnLoad;
	public boolean Back;

	@Override
	public HashMap<String, Class> getMyDefineClass() {
		HashMap<String, Class> map = getNewEmpty();
		map.put("OutFlowPathModel", OutFlowPathModel.class);
		return map;
	}

	@Override
	public HashMap<String, Class> getList() {
		return null;
	}

	@Override
	public String toString() {
		return "OutFlowPathModel [Apply=" + Apply + ", Check=" + Check
				+ ", Give=" + Give + ", Load=" + Load + ", Handover="
				+ Handover + ", UnLoad=" + UnLoad + ", Back=" + Back + "]";
	}

}
